index.md

```diff
@@ -1,91 +1,56 @@
 ---
 layout: default
 title: OLS
-description: A mentoring & training program for Open Science ambassadors
+description: A not-for-profit organisation dedicated to capacity building and diversifying leadership in research worldwide
 image: /images/index.jpg
 photos:
   name: Bérénice Batut
   license: CC BY-SA 4.0
   url: https://flic.kr/p/2gHMJah
+images:
```

Comment:
With the former structure, these images would have been added multiple times across the page. So, rather than having the path repeating everywhere, I put them in the metadata.

We can do without it now, for sure.